### ISSUE TITLE
JSON steps do not match

### ISSUE THREAD

**Issue by gitm3abreak** [dotnet/machinelearning-modelbuilder#1997]
**System Information (please complete the following information):**
 - Model Builder or CLI Version: sentimentModel.mbconfig
 - Visual Studio Version (if applicable): 2022 Community 17.0.5

**Describe the bug**
- On which page of the tutorial did you run into an issue (URL): https://dotnet.microsoft.com/en-us/learn/ml-dotnet/get-started-tutorial/generate-code
- Clear description of the problem: This step "generate-code" does not exist in my installation of VS2022 and so when I tried to compile and run, I had errors.  I had to manually add the portions missing (again, as I did not get the option to generate-code).

**To Reproduce**
Steps to reproduce the behavior: Since the step above does not exist, I pushed the training forward to the "consume" page and copied the sections of code missing (namely, the "using... " and the "string sentiment = ..."

**Expected behavior**
I expected the code to run as-planned without me having to reverse engineer it.  A lesser skilled programmer would had no idea why the code failed to build (although may had notice the discrepancy between the .mbconfig steps in the training and the implementation in the actual model as build.
**Screenshots**
![image](https://user-images.githubusercontent.com/67615394/151702859-fb152e6c-0f80-4ad6-8ccf-f1a5914d287b.png)
Compare the steps above to the steps on the actual training.  Note the missing steps indicated.

**Additional context**

**Comment by beccamc** [comment]
@gitm3abreak Sorry you ran into this problem. You're saying these three files didn't generate as sub items of the .mbconfig file?

![image.png](https://images.zenhubusercontent.com/5d55c3d68fac840001314eaa/16d3282f-11b3-444d-af24-e8bd9b0382e7)

**Comment by gitm3abreak** [comment]
Not exactly.  I meant that the steps shown in the model in the training do not correlalste to the steps seen in my implementation of VS2022.  And because of that, the code generated was not compiling.  I had to manually edit the program.cs to add “using … and the function call at the end 

Fernan

On Jan 31, 2022, at 15:57, Becca McHenry ***@***.***> wrote:

﻿
@gitm3abreak Sorry you ran into this problem. You're saying these three files didn't generate as sub items of the .mbconfig file?



—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
